spring-cloud/spring-cloud-stream/spring-cloud-stream-rabbit/pom.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this need to be a separate module? It seems premature to have a parent/aggregator module with only a single child module. If there are plans to have several spring-cloud-stream modules, then it's ok to leave as-is, otherwise let's just put it all in the spring-cloud-stream module.

Reply:
I think It's more than probably that we have a spring-cloud-stream-kafka for the future. For me, I would let it as it is. Do you agree?

Reply:
I think it's more than probably that we have a module spring-cloud-stream-kafka in the future. I would let it as it is. Do you agree?

Reply:
Yes, I agree we can leave it as-is. This seems to be the way other spring-cloud modules are being organized in the repo.